dist/autosize.js

```diff
@@ -77,6 +77,8 @@
 
 			if (style.boxSizing === 'content-box') {
 				heightOffset = -(parseFloat(style.paddingTop) + parseFloat(style.paddingBottom));
+			} else if (style.boxSizing === 'border-box') {
```

Comment:
This fixes the bug where 
1. multiple lines are added 
2. select and delete all lines

The textarea height would be a few pixels taller than it should be.